### PR TITLE
Comments for Week 3 - Relaxr Part 2

### PR DESCRIPTION
**Pull request by nicollecastrog**
Upon initial glance it seems like you've powered through any issues with clearfix, well done! Code-specific comments coming up. Remember, it might be easier to click on the "Files" tab above and scroll through the comments in there (where the context is more apparent) as opposed to here, on the "Conversation" tab (where you only get a snipped of context).

### CODE REVIEW

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css:15]: I realise the `/* IE */` comment was there in some old code I sent you, so this is totally on me not you, but remove it in future ok? It's misleading as that code is applicable to all browsers, not just Internet Explorer. Thanks! 😸

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  background-color: #f0efef;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css:47]: Well done bringing the Google Font in!

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+nav {
+  float: right;
+  padding-right: 75px;
+}
+
+nav li {
+  font-size: 18px;
+  display: inline-block;
+  padding-right: 30px;
+  padding-top: 28px;
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css:78]: So, I can see a few lines above (on line 58) that you limited the height of the `<header>` to 80px. There's nothing really wrong with that, especially if you're trying to vertically align the children using `vertical-align: middle;`. However `vertical-align: middle;` doesn't work on items that are floated, like the two children of your header (`<h1>` and `<nav>`) are here.

So instead, you might want to consider letting the `<header>` simply be as tall as its children. Which leads me to the comment I'm making about this particular line. As its the two children of `<header>` that you're trying to "pad" and vertically align, it makes sense to add the padding directly to them, and not to _their_ children.

In other words, there's no reason why you shouldn't move this `padding-top` you have here (on line 78) from each individual `<li>` to the parent, the `<nav>`.

Consider this CSS:

```
header {
  background-color: #033048;
}

h1 {
  float: left;
  padding: 12px 0 12px 70px;
  ...etc...
}

nav {
  float: right;
  padding: 28px 75px 28px 0;
}
```

You get a result like this:

image

... I've selected the `<nav>` in the Chrome Dev tools, so you can see the other benefit: by using padding, we can guarantee that the nav is perfectly vertically aligned (via equal amounts of padding `top` and `bottom`) as opposed to working out the maths of `(height of header - height of chosen font of nav) / 2`

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+}
+
+nav li a {
+  text-decoration: none;
+  color: #f9e42e;
+}
+
+main {
+  background-color: #f0efef;
+  width: 80%;
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css:88]: Given that you've limited the `width` of the two children of `<main>` (which are are `<section>` and `<aside>` in pixel values, and the fact that the README specifies a gutter/gap of `20px` between the two columns, you may want to consider using a `width` in `px` here.

Since `<section>` is `700xp` and `<aside>` is 240px and you're meant to have 20px between them, you'll want:

```
main {
  width: 960px;
  margin: 0 auto;
}
```

The result should look like this:

image

... which looks pretty nice, a more "tucked in" design, with less of a gap between the two columns. Additionally, you'll notice that in my proposed CSS for `<main>` above, I've left out `background-color` and that's because you've already set that same background-color on the `<body>` so there's no need to repeat it here

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+  margin: 0 auto;
+}
+
+main section {
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css:92]: As far as I can tell you've only got one `<section>` on this page, but kudos for making use of specificity here!

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+  padding-bottom: 20px;
+}
+
+.article-snippet {
+  padding: 20px 0;
+  font-size: 20px;
+  color: #606161;
+}
+
+article p::first-letter {
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css:111]: Ooooooh fancy!!! Using `:first-letter` instead of a separate `<span>` in the HTML well done!! That said the double `::` is the old way of writing this, you only need a single `:`

Also, since you've given those paragraphs where you want this "large first letter"-effect a class of `article-snippet`, instead of complicating your CSS with references to two html tags, like you do above, you could just reference the class on the `<p>` instead:

`.article-snippet:first-letter { ... }`

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/index.html [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+		</aside>
+	</main>
+
+	<div class="sign-up-now">
+		<a class="button" href="#">Sign Up Now!</a>
+	</div>
+
+	<footer>
+		<h4>Relaxr</h4>
+		<a href="https://www.twitter.com"><img class="some-image" src="./images/twitter.png" alt="twitter logo" /></a>
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/index.html:77]: Proper set up of images that are links, good job! 💯

File: FEWD/Week03/lesson05/homework/week3-relaxr-part2/index.html [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+		<a href="https://www.facebook.com"><img class="some-image" src="./images/facebook.png" alt="facebook logo"/></a>
+		<p>&copy; Copyright 2015. Relaxr</p>
```
Comment (nicollecastrog) [inline FEWD/Week03/lesson05/homework/week3-relaxr-part2/index.html:79]: Using the `&copy;` HTML entity, nice!

**Comment by nicollecastrog** [comment]
Hey Jackie, excellent job on this!! You used a couple of advanced CSS features, it's nice to see you having fun with the assignment! I had a few comments, but they're really minor details, ultimately the assignment looks right, so 💯
